silq/pulses/pulse_types.py

```diff
@@ -667,6 +667,121 @@ def get_voltage(self, t: Union[float, Sequence]) -> Union[float, np.ndarray]:
         return waveform
 
 
+class MultiSinePulse(Pulse):
+    """MultiSinusoidal pulse
+
+    Parameters:
+        name: Pulse name
+        frequencies: list of Pulse frequencies
+        phases: list of Pulse phases
+        amplitudes: list of Pulse amplitudes
```

Comment:
From what I understand, the meaning of amplitudes depends on the interface.
For the arbstudio, these amplitudes correspond to the actual amplitudes of the sine pulses.
For the microwave source, these amplitudes are scaled such that the total amplitude is 1.

This should be clarified in the documentation. Don't refer to specific interfaces, but add that if the interface uses the power, amplitudes are used for IQ modulation and scaled such that the sum is 1. Else the amplitudes are not scaled.

Reply:
done